test/v1/connectors/DeleteBenefitConnectorSpec.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import api.connectors.ConnectorSpec
+import api.models.domain.Nino
+import api.models.outcomes.ResponseWrapper
+import v1.models.request.deleteBenefit.DeleteBenefitRequest
+
+import scala.concurrent.Future
+
+class DeleteBenefitConnectorSpec extends ConnectorSpec {
+
+  private val nino: String      = "AA111111A"
+  private val taxYear: String   = "2019"
```

Comment:
It doesn't affect the test but technically this taxYear is in the wrong format (should be YYYY-YY). I think really we should be using the TaxYear class rather than a string, but this PR is probably big enough!